### PR TITLE
Sent first validation request immediately

### PR DESCRIPTION
**Pull request by rvsia**
Disables `next` button when entering add source wizard

### CODE REVIEW

File: packages/sources/src/addSourceWizard/SourceAddSchema.js [hunk 1]
```diff
@@ -18,8 +18,21 @@ export const asyncValidator = (value, sourceId = undefined) => findSource(value)
     return undefined;
 });
 
+let firstValidation = true;
```
Comment (Hyperkid123) [inline packages/sources/src/addSourceWizard/SourceAddSchema.js]: :eyes:

**Comment by rvsia** [comment]
CI failure is not relates to this PR @Hyperkid123

**Comment by rvsia** [comment]
@Hyperkid123 Please, don't merge this yet, I will investigate disabling the next button by checking 'validating'

**Comment by Hyperkid123** [comment]
@rvsia yeah @karelhala had the same issue yesterday. We still have to wait until the CI works to publish the release properly (or do it manually if it takes to long)

**Pull request merged**